config.example.json

```diff
@@ -14,5 +14,41 @@
         "580736157470556161"
     ],
     "welcomeChannelId": "335964434331860992",
-    "botowners": ["314046311035043840"]
+    "botowners": ["314046311035043840"],
+    "supportChannelId": "247865182477484032",
+    "supportEmbedDescription": [
+        "**Things we do not help with:**",
+        "> • If you are experiencing issues with third party templates or modules, please ask the template authors for support. We cannot assist you with them.",
+        "> • When you are using a pirated template or module. (this will get you banned)",
+        "> • If you are using a very old version of NamelessMC. Newer versions came out with a reason so don't expect support for older versions.",
+        "",
+        "**Before you create a thread:**",
+        "> • `CTRL` + `F` and see if the question has already been answered",
+        "> • See if you can find the information on the documentation (https://docs.namelessmc.com/)",
+        "> • Known issues and workarounds: https://github.com/NamelessMC/Nameless/issues/2822",
+        "",
+        "**Creating a thread in this channel:**",
+        "> Click the button at the bottom of this embed to start the process of creating a thread.",
+        "",
+        "**Some timps to get a quick answer:**",
+        "> • Make sure your issue has a **meaningful title**",
+        ">  ✅ \"*PHPMailer isn't sending emails*\", \"*Error during installation*\", \"*How do I get custom badge colours*\"",
+        ">  ❌ \"Help\", \"My website doesn't work\", \"Hello\"",
+        "> ** **",
+        "> • Provide **all the information** we need to look into the problem",
+        "> - The exact error you see (do not rephrase the error message!) (*including the full stack trace*)",
+        "> - The code you suspect to be the issue and a bit of context",
```

Comment:
We cannot expect 99% of our users to find the code which is causing the issue...